templates/layout/parts/saved_searches_list.html.twig

```diff
@@ -41,15 +41,23 @@
 {% for search in saved_searches %}
    <div class="savedsearches-item grip-savedsearch list-group-item search-line d-flex align-items-center pe-1 {{ active == search['id'] ? 'active' : '' }}"
          data-id="{{ search['id'] }}">
-      <a href="{{ 'SavedSearch'|itemtype_search_path }}?action=load&amp;id={{ search['id'] }}"
```

Comment:
This should target main branch

Reply:
This is just a small condition added. Diff without whitespace is really simple.

Reply:
I guess some part of the PR has been removed since your last comment.

Reply:
I don't think anything changed here since the comment. I thought the comment was about removing the link to the search when it throws an error.